### DISCUSSION TITLE
Gsoc 23

### DISCUSSION THREAD

**Discussion by maheakdave** [alan-turing-institute/sktime#4369] [General]
Hi everyone,
I am a second year student at techno india university, pursuing computer science and engineering with specialization in ai,I am interested in contributing to sktime via gsoc and have selected an idea, deep learning for forecasting, 
I have experience in pytorch and little in tensorflow and any guidance would be appreciated,
Thank you,
Maheak Dave

**Comment by fkiraly** [comment]
IMPORTANT: sktime is not participating in GSoC 2023 this year, also see https://github.com/sktime/sktime/discussions/4046
This should be clear from it not appearing on the orgs page, or being selectable in the application documents (side note, it appears ML frameworks like scikit-learn weren't among the accepted orgs this year more generally).

Kindly be aware of this to avoid preparing an application that you cannot submit.

There will be paid internships (tba soon) and there is the unpaid mentoring programme: https://github.com/sktime/mentoring

**Comment by fkiraly** [comment]
For getting started with contributions, here are "getting started with contributing" guides:

http://www.sktime.net/en/latest/get_involved/contributing.html#contributing
https://github.com/sktime/sktime/issues/1147

**Comment by maheakdave** [comment]
Thank you @fkiraly will go through the links, much appreciated
